### ISSUE TITLE
Update fails

### ISSUE THREAD

**Issue by Abryzz** [ImageMagick/ImageMagick6#226]
### ImageMagick version

6.9.12.83-1

### Operating system

Linux

### Operating system, version and so on

Arch Linux

### Description

Updating ImageMagick6 from the AUR fails due to (probably) a typo.

/usr/include/CL/cl_version.h:22:9: note: '#pragma message: cl_version.h: CL_TARGET_OPENCL_VERSION is not defined. Defaulting to 300 (OpenCL 3.0)'
   22 | #pragma message("cl_version.h: CL_TARGET_OPENCL_VERSION is not defined. Defaulting to 300 (OpenCL 3.0)")
      |         ^~~~~~~
coders/jpeg.c: In function 'WriteJPEGImage_':
coders/jpeg.c:2346:7: error: 'jps_image' undeclared (first use in this function); did you mean 'jps_images'?
 2346 |       jps_image=AppendImages(image,MagickFalse,exception);
      |       ^~~~~~~~~
      |       jps_images
coders/jpeg.c:2346:7: note: each undeclared identifier is reported only once for each function it appears in
coders/jpeg.c:2296:6: warning: unused variable 'jps_images' [-Wunused-variable]
 2296 |     *jps_images = (Image *) NULL,
      |      ^~~~~~~~~~
make[1]: *** [Makefile:8310: coders/jpeg_la-jpeg.lo] Fehler 1
make[1]: *** Es wird auf noch nicht beendete Prozesse gewartet …

### Steps to Reproduce

coders/jpeg.c:2346:7: error: 'jps_image' undeclared (first use in this function); did you mean 'jps_images'?

**Comment by dlemstra** [comment]
Duplicate of #225
